path/utils.go

```diff
@@ -0,0 +1,32 @@
+package path
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// exist checks file or directory existence
+func exist(path string) bool {
+	_, err := os.Stat(path)
+	if err == nil {
+		return true
+	}
+	return os.IsExist(err)
+}
+
+func configPath(pathSuffix string) string {
+	return pathSelector(UserConfig, SystemConfig, pathSuffix)
+}
+
+func libexecPath(pathSuffix string) string {
+	return pathSelector(UserLibexec, SystemLibexec, pathSuffix)
+}
+
+func pathSelector(userPath, systemPath, pathSuffix string) string {
```

Comment:
Can you add a comment saying this will change once we finalize https://github.com/notaryproject/notation/issues/203

Reply:
comment in base.go:83